db/migrate/20210518074409_create_reporting_patient_blood_pressures_per_month.rb

```diff
@@ -0,0 +1,5 @@
+class CreateReportingPatientBloodPressuresPerMonth < ActiveRecord::Migration[5.2]
+  def change
+    create_view :reporting_patient_blood_pressures_per_month, materialized: true
```

Comment:
We wanted to prefix all reporting tables with `reporting_`, since the kind of data, and the context in which we are using them becomes clear immediately, especially since they are colocated with the app tables right now. Furthermore, the reporting tables currently use a different timezone from the app tables.

The rationale for the `patient_` prefix, is to allow for a `facility_` version of the same data which will contain facility wise aggregates. It's a bit forward thinking, but we think it's worth the few extra key strokes.

-- @harimohanraj89 and @ssrihari